### ISSUE TITLE
Support WebSockets

### ISSUE THREAD

**Issue by davidbrochart** [simonw/asgi-proxy-lib#3]
Nice project, thanks for sharing!
Correct me if I'm wrong, but it doesn't seem to support WebSockets? If not, it could be interesting to use [httpx-ws](https://github.com/frankie567/httpx-ws).

**Comment by simonw** [comment]
No, it doesn't support websockets yet - that would be a useful addition, since then you could do things like have `/ws` on an existing ASGI application proxy through to some other host.
